index.js

```diff
@@ -1,19 +1,22 @@
 var parse = require('./parser').parse;
+var print = require('./printer').print;
 
 var cache = {};
 
-function stripLoc (doc) {
+function stripLoc (doc, removeRoot) {
```

Comment:
This argument isn't well named - I think it should either be "isRoot" or "shouldRemoveThisLevel" or something.